src/App.js

```diff
@@ -65,7 +63,7 @@ class App extends Component {
     })
   }
 
-  checkAnswer(answerId) {
+  checkAnswer = answerId => {
     if (answerId === this.state.currentTrack.track.id) {
       swal('Bravo', 'Vous avez la boonne réponse', 'success')
```

Comment:
'success' is not a label, it is a value from an enum used by swal.

Check the following Theodo Code Principle: https://github.com/theodo/theodo-code-principles/blob/master/README.md#no-static-strings

Check if swal provides an object of constants.